symbols/overlay29.yml

```diff
@@ -1085,6 +1085,40 @@ overlay29:
         r2: y position
         r3: flags
         stack[0]: visibility flag
+    - name: PrepareTrapperTrap
+      address:
+        NA: 0x22EDBD4
+      description: |-
+        Saves the relevant information in the dungeon struct to later place a trap at the
+        location of the entity. (Only called with trap ID 0x19, but could be used with others).
+        
+        r0: entity pointer
+        r1: trap ID
+        r2: team (see struct trap::team)
+    - name: TrySpawnTrap
+      address:
+        NA: 0x22EDCBC
+      description: |-
+        Checks if the a trap can be placed on the tile. If the trap ID is above 0x18 (the
+        last value for a trap), randomly select another trap (except for wonder tile). After
+        30 failed attempts to select a non-wonder tile trap ID, default to chestnut trap.
+        If the checks pass, spawn the trap.
+        
+        r0: position
```

Comment:
This parameter doesn't line up with the function signature. Can we fix whichever one is wrong?